actionpack/lib/action_dispatch/http/request.rb

```diff
@@ -84,7 +84,8 @@ def controller_class
         PASS_NOT_FOUND
       end
     end
-
+    #key? method returns a boolean value
+    #returns true if 'key'(the parameter) has been set accordingly
```

Comment:
Thanks a lot @eileencodes.
I am new to the open source community and will surely keep your message in mind in the future.
I have created a new pull request following your instructions.The link is: https://github.com/rails/rails/pull/27482